v63003/u_inref/ydb297.csh

```diff
@@ -0,0 +1,35 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+# Tests that when over 31 locks hash to the same value, LOCK commands work correctly
+#
+$gtm_tst/com/dbcreate.csh mumps 1 >>& dbcreate.out
+# Simulates 2 errors in prior versions, either inappropriately seizes ownership of a lock or test hangs
+# Randomly chooses an option (1 will cause the test to hang, 0 will cause locks to be inappropriately seized)
+setenv hang `$gtm_tst/com/genrandnumbers.csh 1 0 1`
+$ydb_dist/mumps -run ydb297^ydb297 >>& job.out
```

Comment:
You will need to add a "unsetenv ydb_lockhash_n_bits" before this line. This will make sure the test framework (which randomizes that env var) and code do not artificially cause further hash collisions than this test expects (which would cause test failures).